inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/kafka/KafkaFederationWorker.java

```diff
@@ -117,7 +112,7 @@ public void run() {
             } catch (Exception e) {
                 LOG.error(e.getMessage(), e);
                 if (tx != null) {
-                    tx.rollback();
+                    tx.commit();
```

Comment:
When it is fail to send data to Kafka, here need to rollback transaction.

Reply:
fixed